packages/playground/src/components/select_vm_image.vue

```diff
@@ -84,14 +84,20 @@ watch(
 );
 
 function isFlistExist(flist: string) {
-  return fetch(flist + ".md5")
-    .then(res => res.status !== 200)
-    .then(invalid => {
-      if (invalid) throw new Error();
-    })
-    .catch(() => {
-      return { message: `Couldn't find flist with the provided url.`, isExist: false };
-    });
+  if (flist.includes(".flist")) {
+    return fetch(flist + ".md5")
+      .then(res => res.status !== 200)
+      .then(invalid => {
+        if (invalid) throw new Error();
+      })
+      .catch(() => {
+        return { message: `Couldn't find flist with the provided url.`, isExist: false };
+      });
+  } else {
```

Comment:
these two branches don't return the same type on is a promise and the other is an object

Reply:
I know, I'm working on fixing that